doc/news/_preparation_next_release.md

```diff
@@ -415,7 +415,7 @@ This section keeps you up-to-date with the multi-language support provided by El
 
 ## Tests
 
-- <<TODO>>
+- Fix an Issue where scripts/dev/fix-spelling does not work, if a resolved path contains whitespaces _(Juri Schreib @Bujuhu)_ _(Nikola Prvulovic @Dynamichost96)_
```

Comment:
Please also write that the file was renamed. Please also fix occurrence within the same file (`https://master.libelektra.org/scripts/sed`). Also doc/news/2022-03-10_0.9.9.md and doc/news/2017-12-22_0.8.21.md and tests/shell/check_spelling.sh have references.